### PR TITLE
Updated Sign In With Ethereum Docs With Video Guide and Texts

### PR DESCRIPTION
**Pull request by SolomonFoskaay**
I believe beginners comprehend video faster than text. Thus, have added a video from the official unlock youtube channel that aligns with the Sign In with Ethereum to spice up the docs. If approved, then will consider replicating the same for other aspects of the docs with existing videos on the channel (or when the video is done in the future).

### CODE REVIEW

**Comment by ccarfi** [comment]
thanks, @SolomonFoskaay! have added this to the review queue. thank you so much for the contribution!

File: docs/tools/sign-in-with-ethereum/README.md [hunk 1]
```diff
@@ -10,8 +10,15 @@ In many contexts, your application does not need a full "web3" provider, but jus
 
 > "_Ethereum_" here does not refer to any network in particular but to the type of wallet that can used. Since Polygon, Gnosis Chain or Optimism for example are all using the same wallet, you can (should!) absolutely use the same "Sign In with Ethereum".
 
+> Yeah, the sign in with Ethereum seems confusing to make you think it's all about Ethereum only. No, it covers Web3 sign in as a whole not just Ethereum (though initially, it all started with Ethereum - no wonder the confusing naming).
+
```
Comment (wonderwomancode) [inline docs/tools/sign-in-with-ethereum/README.md:14]: I love this change in language to be more friendly. As a rule we always say web3 with a little "w" for consistency.

**Review by wonderwomancode — APPROVED** [review]

**Comment by SolomonFoskaay** [comment]
> @SolomonFoskaay Could you please fix the PR? It looks like one of the images is missing... Thank you!

@julien51 I checked and the image is working on my end. I do not see where it's broken to be able to fix it, please can you point me to it?

Also, @wonderwomancode seems you have made the change to the lowercase 'w' and did you spot the broken image @julien51 mentioned too?

Will like to know how to have any pending issues fixed.
Thanks!

**Comment by wonderwomancode** [comment]
@SolomonFoskaay yeah I was using my mobile this morning it doesn't look like my final comment was recorded regarding the link that needs to be moved into /src/static/img and you can see we additionally use img subdirectories that match the file structure of the pages. You can see how the links in the code look inside one of the other pages but essentially you don't put the entire path but start with /img.

**Comment by julien51** [comment]
Check the build @SolomonFoskaay :) you will see the error.

File: docs/tools/sign-in-with-ethereum/README.md (hunk 1 above)
Comment (SolomonFoskaay) [reply docs/tools/sign-in-with-ethereum/README.md:14]: I have changed the Web3 to web3 as requested

**Comment by SolomonFoskaay** [comment]
@julien51 @wonderwomancode 
PR Issues Resolved:
1) Web3 changed to web3 for consistency
2) Created and moved the image inside the appropriate folder "sign-in-with-ethereum" folder inside /img/tools/
3) Build broken image error resolved
Thanks!

**Review by julien51 — APPROVED** [review]

**Comment by julien51** [comment]
Let's have @wonderwomancode give the final check but this looks very good!

**Pull request merged**